### ISSUE TITLE
Loading compiled LLDP-MIB fails with error "'lldpLocManAddrEntry' is not defined"

### ISSUE THREAD

**Issue by AlexLardschneider** [lextudio/pysnmp#91]
### Expected behavior

Trying to load the compiled version of LLDP-MIB (found [here](https://raw.githubusercontent.com/lextudio/mibs.pysnmp.com/master/asn1/LLDP-MIB)) and compiled using

```
mibdump --destination-directory=compiled --rebuild --quiet ./mibs/common/*
```

fails with the following error:

```
Traceback (most recent call last):
  File "/Users/Alex/Developer/Test/venv/lib/python3.11/site-packages/pysnmp/smi/builder.py", line 376, in loadModule
    exec(codeObj, g)
  File "compiled/LLDP-MIB.py", line 485, in <module>
    lldpLocManAddrEntry.registerAugmentions(
    ^^^^^^^^^^^^^^^^^^^
NameError: name 'lldpLocManAddrEntry' is not defined. Did you mean: 'lldpConfigManAddrEntry'?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.11/3.11.9/Frameworks/Python.framework/Versions/3.11/lib/python3.11/threading.py", line 1045, in _bootstrap_inner
: INFO     : aiomq.py         : RabbitMQ: Starting publisher.
    self.run()
  File "/usr/local/Cellar/python@3.11/3.11.9/Frameworks/Python.framework/Versions/3.11/lib/python3.11/threading.py", line 982, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/Alex/Developer/Test/app.py", line 87, in init_snmp
    mib_builder.init()
  File "/Users/Alex/Developer/Test/util/mib_builder.py", line 121, in init
    mibBuilder.loadModules(*MIBS_GENERIC)
  File "/Users/Alex/Developer/Test/venv/lib/python3.11/site-packages/pysnmp/smi/builder.py", line 417, in loadModules
    self.loadModule(modName, **userCtx)
  File "/Users/Alex/Developer/Test/venv/lib/python3.11/site-packages/pysnmp/smi/builder.py", line 380, in loadModule
    raise error.MibLoadError(
pysnmp.smi.error.MibLoadError: MIB module 'compiled/LLDP-MIBcompiled/LLDP-MIB.py' load error: ['Traceback (most recent call last):\n', '  File "/Users/Alex/Developer/Test/venv/lib/python3.11/site-packages/pysnmp/smi/builder.py", line 376, in loadModule\n    exec(codeObj, g)\n', '  File "compiled/LLDP-MIB.py", line 485, in <module>\n    lldpLocManAddrEntry.registerAugmentions(\n    ^^^^^^^^^^^^^^^^^^^\n', "NameError: name 'lldpLocManAddrEntry' is not defined\n"] caused by <class 'NameError'>: name 'lldpLocManAddrEntry' is not defined
```

It looks like the compiled version of LLDP-MIB is trying to call the function `registerAugmentions` on object `lldpLocManAddrEntry` (like 485), but `lldpLocManAddrEntry` is first defined on line 819.

### Actual behavior

PySNMP should load the compiled MIB file without errors.

### Detailed steps

See above.

### Python package information

```
pysmi-lextudio==1.4.3
pysnmp-lextudio==6.2.1
pyasn1==0.6.0
```

### Operating system information

macOS

### Python information

3.11

**Comment by steven-douilliet** [comment]
See also #90 which talks about LLDP-MIB.

**Comment by lextm** [comment]
This all comes down to how PySMI compiles RMON2-MIB and LLDP-MIB.

While the bugfix won't come sooner as we have other priorities at hand, the workaround is simple enough,

1. Open up `LLDP-MIB.py` in a text editor.
2. Find the lines that refer to `lldpLocManAddrEntry` (two sections in all), cut and paste them right behind `lldpLocManAddrEntry`'s definition line.

**Comment by lextm** [comment]
Close this now as won't work on it in near future.

**Comment by lextm** [comment]
Should have been fixed in PySMI release 1.5.
